cms-xcache/xcache-consistency-check-wrapper.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+ENABLE=`cat /etc/environment |  grep XC_XCACHE_CONSISTENCY_CHECK | awk -F "=" '{print $2}'`
```

Comment:
```suggestion
ENABLE=`awk -F "=" '/^XC_XCACHE_CONSISTENCY_CHECK/ {print $2}' /etc/environment`
```
This assumes that each line in the environment file is `VAR=VAL`